gulpfile.js

```diff
@@ -211,6 +201,106 @@ function buildIE() {
   return convertForIE();
 }
 
+function lintCSSFunctions() {
+  const validCSSFunctions = [
+    // https://www.w3schools.com/cssref/css_functions.asp
+    'attr',
+    'calc',
+    'cubic-bezier',
+    'hsl',
+    'hsla',
+    'linear-gradient',
+    'radial-gradient',
+    'repeating-linear-gradient',
+    'rgb',
+    'rgba',
+    'var',
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function
+    'matrix',
+    'matrix3d',
+    'perspective',
+    'rotate',
+    'rotate3d',
+    'rotateX',
+    'rotateY',
+    'rotateZ',
+    'scale',
+    'scaleX',
+    'scaleY',
+    'scaleZ',
+    'skew',
+    'skewX',
+    'skewY',
+    'translate',
+    'translate3d',
+    'translateX',
+    'translateY',
+    'translateZ',
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
+    'inset',
+    'circle',
+    'ellipse',
+    'polygon',
+    'path',
+    'rect',
+    // Custom
+    'blur',
+    'counter',
+    'repeat',
+    'minmax',
+    'url'
+  ];
+  const regex = /:\s+(\w[\w\d]+)\(.*\)/g;
+  let failed = false;
+
+  return src('./dist/**/*.css')
+    .pipe(
+      through2.obj((chunk, _, cb2) => {
+        const css = chunk.contents.toString();
+
+        let result;
+        // eslint-disable-next-line no-cond-assign
+        while ((result = regex.exec(css))) {
+          if (!validCSSFunctions.includes(result[1])) {
```

Comment:
are you ignoring case? My regex is not good enough to tell :) I think CSS functions are not case sensitive

Reply:
Nice catch! I hope `node-sass`'s compiled output would never have to put this to the test, though...